responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -471,6 +471,12 @@ def _validate_rai_insights_input_parameters(
                     columns=[target_column], axis=1)
                 small_test_data = small_test_data.drop(
                     columns=[target_column], axis=1)
+                if len(small_train_data.columns) == 0 or \
+                        len(small_test_data.columns) == 0:
+                    raise UserConfigValidationException(
+                        'There is no feature in the dataset,'
+                        ' or all features have been dropped from the dataset'
+                    )
```

Comment:
Can you check for specific condition and raise different exceptions? Raise one exception for no features and one for all features being part of dropped features?

Reply:
Updated!